Syntaxes/Asciidoctor.sublime-syntax

```diff
@@ -2202,34 +2207,11 @@ contexts:
   #     double colon followed by a space inside a label, i.e. it matches the
   #     *last* double colon, not the first. I don't know how to do that
   #     *effectively.
-
-  # >>>>>>>>>>>>>>>>>>>>>>>>>>>
-  # dlist_item_label:
-  #   comment: |
-  #     Label of a definition (labeled) list item.
-  #
-  #     Examples:
-  #       Label level 1:: lorem ipsum
-  #       Label level 2::: dolor sit amet
-  #       Label level 3:::: consectetur
-  #       Label level 1::
-  #         lorem ipsum
-  #       Another label :: lorem ipsum
-  #       Last::label:: dolor sit amet
-  #
-  #     Note: This rule is not strictly correct, because Asciidoctor allows
-  #     double colon followed by a space inside a label, i.e. it matches the
-  #     *last* double colon, not the first. I don't know how to do that
-  #     *effectively.
-  #   name: markup.list.labeled.asciidoc
-  #   contentName: meta.list.label.asciidoc
-  #   begin: ^\s*(?=.*:{2,4}(?:\s|$))
-  #   end: (:{2,4})(?:\s|$\n?)
-  #   endCaptures:
-  #     '1': {name: constant.labeledlist.separator.asciidoc}
-  #   - include: inline
-  #   - include: characters
-  # <<<<<<<<<<<<<<<<<<<<<<<<<<<
+  #AR: why isn't this working?
```

Comment:
This change is also present in PR #14:

https://github.com/tajmone/ST3-Asciidoctor/pull/13/files#diff-295ccf9dd4a32466da41a1f954891029454e286e2655738787f6f04f07626a45R2210

All PRs should be from branches based on the `dev` branch, not from branches of other PRs, otherwise if you apply changes to the other PRs, these duplicate changes will result not only in redundant changes, but also in conflicts.

Reply:
Suggested action: in this PR branch, delete all unrelated files and amend head commit, rebase on `dev` branch, then force push. Even if the target PR branch here is `master` (and we can't change that), we can at least start fixing some issues before re-creating this PR against `dev`.